AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.spec.ts

```diff
@@ -464,6 +467,13 @@ describe('HearingScheduleComponent returning to page', () => {
         expect(component.courtAddressControl.value).toBe(existingRequest.hearing_venue_id);
         expect(component.multiDaysHearing).toBe(existingRequest.isMultiDayEdit);
     });
+
+    it('should set controls for venue', () => {
+        component.ngOnInit();
+        expect(component.selectedCourtName).toBeTruthy();
+        expect(component.selectedCourtCode).toBeTruthy();
```

Comment:
Is should assert the values, 'Bedford' and '333'

Reply:
test modified